js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -195,6 +192,22 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating shareInfo function to MiniAppBridge object.
+   * This function does not return anything back on success.
+   * @param {info} The shared info object.
+   */
+  shareInfo(info: ShareInfoType) {
+    return new Promise<undefined>((resolve, reject) => {
+      return this.executor.exec(
+        'shareInfo',
+        info,
+        success => resolve(undefined),
```

Comment:
Do you think we can return something like a `SUCCESS` would be better?  We can return SUCCESS once we pass the string to the Host app delegate/callback. What do you think ?